components/ReviewTransactionModal.tsx

```diff
@@ -0,0 +1,154 @@
+import useNotifications from "@/hooks/useNotifications";
+import useRailgunTx from "@/hooks/useRailgunTx";
+import useShieldPrivateKey from "@/hooks/useShieldPrivateKey";
+import { shortenAddress } from "@/utils/address";
+import { feePercent } from "@/utils/constants";
```

Comment:
Lots of unused imports/vars in this file and some others

Reply:
Going to add eslint:recommended to the eslint.json so we catch these in the build